### ISSUE TITLE
Missing vanilla mechanics

### ISSUE THREAD

**Issue by bobucles** [CorsixTH/CorsixTH#1454]
#### Describe the issue
1. Pawns do not evacuate during "place a door" stage of blueprint. This happens in vanilla even though the "zzz" isn't shown.

2. Epidemic clock does not instantly end when an epidemic is successfully eradicated. This is the leading cause of an early clock ending, at least in a good hospital.

#### System Information
CorsixTH Version: 0.62
Windows 10*
Theme Hospital Version: GOG.com

**Comment by TheCycoONE** [comment]
Can you split this into two issues so we can track each please.

**Comment by TheCycoONE** [comment]
I'm making this ticket about your first issue.

**Comment by ARGAMX** [comment]
Can be closed because
second issue moved to more specialised ticket too
#2738

**Comment by lewri** [comment]
Is the first issue actually fixed by another PR?

**Comment by ARGAMX** [comment]
> Is the first issue actually fixed by another PR?

oops, indeed...
I was read "making ticket" as creating new one. 
Now I realised that this was just ticket renaming and that's all.

I can record a video for the first problem and create more specified ticket.
Should I do?

**Comment by lewri** [comment]
This one doesn't seem to actually be a big problem overall. In TH, humanoids were 'hinted' at doors phase to start getting off the tiles. In CorsixTH we lack the hinting and instead have an explicit kick off after doors.

It could be simple enough to call `EditRoom:clearArea()` without the lockdown and cursor change when we go to enter the doors phase. Not tested though.

**Comment by ARGAMX** [comment]
I think the point of this task is to start expelling humanoids from the construction site before placing the door, not after.
That is, when the blueprints change their transparency to less transparent, the humanoids should already be given the command to leave the construction site.
This is exactly what happens in the original TH.
In CorsixTH, humanoids are currently expelled only after placing the door and attempting to click the checkbox.
This means TH starts the construction site clearing process earlier than CorsixTH.

Moreover, humanoids do not leave the construction site via the optimal route, but I dedicated #3505 to this certain issue.
